_spinnaker/kubernetes_deployments.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: Kubernetes Deployments
+order: 132
+---
+{:.no_toc}
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
+
+This document focuses on the manifest based (aka V2) Kubernetes provider.
+
+## Kubernetes Deployments
+Armory Spinnaker delegates the deployment of containers to Kubernetes. Kubernetes then proceeds with a rolling update deployment which is effectively a rolling blue/green deployment.
+Pods are created in batches and when a pod is deemed [healthy](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/) it starts receiving traffic.
+
+### Example
+We've defined a simple pipeline where we first define an artifact (a Docker image) with a version coming from a pipeline parameter.
+You'd usually get the image from a trigger via container registry or a Jenkins job with the same result.
+
+![image](https://cl.ly/b93f96f9cf44/%255Bf1c1b353770f4fcb2ef259a94c46a9ca%255D_Image%2525202018-09-27%252520at%2525204.47.42%252520PM.png)
+
+
+The second step defines the deployment of that image. It's a simple "*Deploy (Manifest)*" stage. Here we're adding the static deployment manifest via a text field but you'd usually retrieve it as an artifact directly
+or via a [Helm bake stage](https://kb.armory.io/kubernetes/using-spinnaker-and-helm/).
+
+![image](https://cl.ly/912a0bfea50a/%255Bb2347374090ddd8ccff9178ac0cee09a%255D_Image%2525202018-09-27%252520at%2525206.01.23%252520PM.png)
+
+As a matter of fact the deployment manifest is not entirely static, Spinnaker will replace the image name with the actual tagged name from the bound artifact.
```

Comment:
Minor grammar notes:  Should add a comma after "As a matter of fact" and and use either a dash or semicolon after "static":  "As a matter of fact, the deployment manifest is not entirely static -- Spinnaker will replace the image name with..."

Reply:
Fixed